prow/config/config.go

```diff
@@ -224,6 +229,15 @@ func parseConfig(c *Config) error {
 	if c.PodNamespace == "" {
 		c.PodNamespace = "default"
 	}
+
+	level := logrus.InfoLevel
+	if lvl, err := logrus.ParseLevel(c.LogLevel); err != nil {
+		logrus.Warningf("Cannot parse log level, will default to 'info': %v", err)
```

Comment:
Propagate up the error rather than allowing it by, and special-case empty string to mean `logrus.InfoLevel`.

Reply:
done